kernel/kernel-api/src/main/java/io/delta/kernel/internal/rowtracking/RowTracking.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.internal.rowtracking;
+
+import static io.delta.kernel.internal.util.Preconditions.checkArgument;
+
+import io.delta.kernel.data.Row;
+import io.delta.kernel.internal.DeltaErrors;
+import io.delta.kernel.internal.SnapshotImpl;
+import io.delta.kernel.internal.TableFeatures;
+import io.delta.kernel.internal.actions.*;
+import io.delta.kernel.utils.CloseableIterable;
+import io.delta.kernel.utils.CloseableIterator;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicLong;
+
+/** A collection of helper methods for working with row tracking. */
+public class RowTracking {
+  private RowTracking() {
+    // Empty constructor to prevent instantiation of this class
+  }
+
+  private static final int ADD_FILE_ORDINAL = SingleAction.FULL_SCHEMA.indexOf("add");
```

Comment:
Instead, please just make `SingleAction::ADD_FILE_ORDINAL` be `public` instead of `private`, and use that here! Less code duplication!